components/brave_new_tab_ui/containers/newTab/gridTile.tsx

```diff
@@ -53,46 +49,17 @@ class TopSite extends React.PureComponent<Props, {}> {
       <Tile
         title={siteData.title}
         tabIndex={0}
-        style={{
-          // Visually inform users that dragging a pinned site is not allowed.
-          cursor: isGridSitePinned(siteData) ? 'not-allowed' : 'grab'
-        }}
       >
-        <TileActionsContainer>
-          <TileAction onClick={this.onTogglePinnedTopSite.bind(this, siteData)}>
-            {isGridSitePinned(siteData) ? <PinIcon /> : <PinOIcon />}
-          </TileAction>
-          {
-            // Disallow removing a pinned site
-            isGridSitePinned(siteData)
-              ? (
-                <TileAction
-                  style={{ opacity: 0.25, cursor: 'not-allowed' }}
-                >
-                  <CloseStrokeIcon />
-                </TileAction>
-              ) : (
-                <TileAction
-                  onClick={this.onIgnoredTopSite.bind(this, siteData)}
-                >
-                  <CloseStrokeIcon />
-                </TileAction>
-              )
-          }
-        </TileActionsContainer>
         {
-          // Add the permanent pinned icon if site is pinned
-          isGridSitePinned(siteData)
-            ? (
-              <TileAction
-                onClick={this.onTogglePinnedTopSite.bind(this, siteData)}
-                standalone={true}
-              >
-                <PinIcon />
+          !siteData.defaultSRTopSite
+          ? <TileActionsContainer>
+              <TileAction onClick={this.onIgnoredTopSite.bind(this, siteData)}>
```

Comment:
minor nit: as far as I can see, this doesn't need to bind a new function every time we render since `siteData` is available on `this.props` inside the `onIgnoredTopSite` class function.